profiles/updates/4Q-2024

```diff
@@ -4,3 +4,4 @@ slotmove media-sound/kmetronome 5 0
 move net-im/sendxmpp-amm net-im/sendxmpp-scala
 move dev-python/jinja dev-python/jinja2
 move dev-python/gmpy dev-python/gmpy2
+move dev-java/jakarta-annotations-api dev-java/jakarta-annotation-api
```

Comment:
> i just tried this commit and i think this change in fact causes breakage in the tree:
> 
> ```
> # PORTDIR=/usr/src/gentoo.git/ FEATURES=test emerge -va1t cdi-api memoryfilesystem
> 
> Performing Global Updates
> (Could take a couple of minutes if you have a lot of binary packages.)
>   .='update pass'  *='binary update'  #='/var/db update'  @='/var/db move'
>   s='/var/db SLOT move'  %='binary move'  S='binary SLOT move'
>   p='update /etc/portage/package.*'
> /usr/src/gentoo.git/profiles/updates/4Q-2024.......
> @##
> 
> 
> These are the packages that would be merged, in reverse order:
> 
> Calculating dependencies... done!
> Dependency resolution took 71.59 s (backtrack: 0/20).
> 
> [ebuild  N     ] dev-java/memoryfilesystem-2.3.0-r1::gentoo  USE="-doc -source" 125 KiB
> [ebuild     U  ] dev-java/cdi-api-4.0.1-r4::gentoo [4.0.1-r3::gentoo] USE="test* -doc -source" 0 KiB
> ```

Whatever I tried, I cannot reproduce that error. With `PORTDIR=/var/db/repos/gentoo` (Yours doesn't work on my systems) it always pulls the new `dev-java/jakarta-annotation-api` as another `[ebuild  N     ]`.
And that's independent of having changed `profiles/updates/4Q-2024` or not, so I think it might be safe to omit that change. But for the documentation, I'm not sure.